src/System.Drawing.Common/src/System/Drawing/Icon.cs

```diff
@@ -20,8 +18,7 @@ namespace System.Drawing
     ///     represent an object.  Icons can be thought of as transparent bitmaps, although
     ///     their size is determined by the system.
     /// </devdoc>
-    [Serializable]
-    public sealed class Icon : MarshalByRefObject, ISerializable, ICloneable, IDisposable
+    public sealed partial class Icon : MarshalByRefObject, ICloneable, IDisposable
     {
 #if FINALIZATION_WATCH
```

Comment:
I've put it on my checklist in https://github.com/dotnet/corefx/issues/20706. I'm going to start going through those one-by-one when I finish the remaining test PR's.